packages/currency/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@requestnetwork/currency",
+  "version": "0.1.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Currency tools for Request Network packages.",
+  "keywords": [
+    "requestnetwork",
+    "currency",
+    "utils"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/RequestNetwork/requestNetwork.git"
+  },
+  "homepage": "https://github.com/RequestNetwork/requestNetwork/tree/master/packages/currency#readme",
+  "bugs": {
+    "url": "https://github.com/RequestNetwork/requestNetwork/issues"
+  },
+  "license": "MIT",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "directories": {
+    "lib": "src",
+    "test": "test"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc -b",
+    "clean": "shx rm -rf dist tsconfig.tsbuildinfo",
+    "lint": "tslint --project . && eslint \"src/**/*.ts\"",
+    "lint-staged": "lint-staged",
+    "prepare": "yarn run build",
+    "test": "jest",
+    "test:watch": "yarn test --watch"
+  },
+  "dependencies": {
+    "@metamask/contract-metadata": "1.20.0",
+    "@requestnetwork/types": "0.29.2",
+    "ethers": "4.0.48"
+  },
+  "devDependencies": {
+    "@types/jest": "26.0.13",
+    "@typescript-eslint/parser": "4.1.1",
+    "eslint": "7.9.0",
+    "eslint-plugin-spellcheck": "0.0.17",
+    "eslint-plugin-typescript": "0.14.0",
+    "jest": "26.4.2",
+    "lint-staged": "10.3.0",
+    "mocha": "8.1.3",
+    "nyc": "15.1.0",
```

Comment:
Those 2 are no longer used. We should remove them from `utils` too, it's a mistake that they're still there.

Reply:
@benjlevesque What about the file .nycrc?